lib/configuration_selector.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'board'
+require_relative 'game_builder'
+require_relative 'player_selector'
+require_relative 'order_selector'
+require_relative 'players_builder'
+
+class ConfigurationSelector
+  def initialize(console)
+    @console = console
+  end
+
+  def select_options
+    @console.player_menu
+    player_names = choose_players
+
+    @console.order_menu(player_names)
+    ordered_players = choose_order(player_names)
+
+    board = Board.new
+    markers = %w[X O]
+
+    players = PlayersBuilder.build(ordered_players, markers)
```

Comment:
instead of initializing players here you should do it inside the game builder.  something like `builder.set_players(ordered_players, markers)`

Reply:
sorry I just pushed up new changes based on Brian's comments so code is no longer current